kafkajobs/opts.go

```diff
@@ -91,6 +92,10 @@ type SASL struct {
 	UserAgent    string `mapstructure:"user_agent" json:"user_agent"`
 }
 
+type Ping struct {
+	Timeout time.Duration `mapstructure:"timeout" json:"timeout"`
```

Comment:
Let's do this mandatory, cause I think this check is kinda important to start the driver. Let's do the following (in the `config.go`, inside `conf.InitDefault()`):
```go
if c.Ping == nil {
    c.Ping = &Ping {
        // default - 10 seconds 
        Timeout: time.Second*10,
    }
}
```

And thus we don't need to use the `if conf.Ping != nil {` in the `From*` methods.

Reply:
I think 10 seconds should be enought to ping the Kafka broker, WDYT?

Reply:
Unfortunately, 10 seconds may not be enough if the Kafka is external.

Reply:
10 second latency to reach the broker?

Reply:
Yes, if the connection to the external cluster is very slow.

Reply:
Hm, never had this issue before... But ok, let it be 1 minute. `time.Minute`.

Reply:
I recently pressed 10 seconds as the default timeout. This is a good time, in most cases this is enough. If the ping lasts more than 10 seconds, then the developer himself must intentionally increase the timeout.

I think it will be convenient if it is possible to set the timeout.